stdlib/src/builtin/builtin_list.mojo

```diff
@@ -138,16 +138,19 @@ struct VariadicList[type: AnyRegType](Sized):
         return __mlir_op.`pop.variadic.size`(self.value)
 
     @always_inline
-    fn __getitem__(self, index: Int) -> type:
+    fn __getitem__[indexer: Indexer](self, idx: indexer) -> type:
```

Comment:
style: here and elsewhere: we try to make type parameters PascalCase, could you please change these?
```suggestion
    fn __getitem__[IndexerType: Indexer](self, idx: IndexerType) -> type:
```
If you want, I'm also okay with calling these just `T` for brevity.